sdk/elixir/lib/dagger/gen/container.ex

```diff
@@ -1,1143 +1,1050 @@
+      |> maybe_put_arg("target", optional_args[:target])
+      |> maybe_put_arg("buildArgs", optional_args[:build_args])
+      |> maybe_put_arg(
+        "secrets",
+        if(optional_args[:secrets],
+          do: Enum.map(optional_args[:secrets], &Dagger.ID.id!/1),
+          else: nil
+        )
       )
 
-      %Dagger.Container{selection: selection, client: container.client}
-    end
-  )
-
-  (
-    @doc "Retrieves the container with the given directory mounted to /.\n\n## Required Arguments\n\n* `directory` - Directory to mount."
-    @spec with_rootfs(t(), Dagger.Directory.t()) :: Dagger.Container.t()
-    def with_rootfs(%__MODULE__{} = container, directory) do
-      selection = select(container.selection, "withRootfs")
-
-      (
-        {:ok, id} = Dagger.Directory.id(directory)
-        selection = arg(selection, "directory", id)
+    %Dagger.Container{
+      selection: selection,
+      client: container.client
+    }
+  end
+
+  @doc "Retrieves default arguments for future commands."
+  @spec default_args(t()) :: {:ok, [String.t()]} | {:error, term()}
+  def default_args(%__MODULE__{} = container) do
+    selection =
+      container.selection |> select("defaultArgs")
+
+    execute(selection, container.client)
+  end
+
+  @doc """
+  Retrieves a directory at the given path.
+
+  Mounts are included.
+  """
+  @spec directory(t(), String.t()) :: Dagger.Directory.t()
+  def directory(%__MODULE__{} = container, path) do
+    selection =
+      container.selection |> select("directory") |> put_arg("path", path)
+
+    %Dagger.Directory{
+      selection: selection,
+      client: container.client
+    }
+  end
+
+  @doc "Retrieves entrypoint to be prepended to the arguments of all commands."
+  @spec entrypoint(t()) :: {:ok, [String.t()]} | {:error, term()}
+  def entrypoint(%__MODULE__{} = container) do
+    selection =
+      container.selection |> select("entrypoint")
+
+    execute(selection, container.client)
+  end
+
+  @doc "Retrieves the value of the specified environment variable."
+  @spec env_variable(t(), String.t()) :: {:ok, String.t() | nil} | {:error, term()}
+  def env_variable(%__MODULE__{} = container, name) do
+    selection =
+      container.selection |> select("envVariable") |> put_arg("name", name)
+
+    execute(selection, container.client)
+  end
+
+  @doc "Retrieves the list of environment variables passed to commands."
+  @spec env_variables(t()) :: {:ok, [Dagger.EnvVariable.t()]} | {:error, term()}
+  def env_variables(%__MODULE__{} = container) do
+    selection =
+      container.selection |> select("envVariables") |> select("id")
+
+    with {:ok, items} <- execute(selection, container.client) do
+      {:ok,
+       for %{"id" => id} <- items do
+         %Dagger.EnvVariable{
+           selection:
+             query()
+             |> select("loadEnvVariableFromID")
+             |> arg("id", id),
+           client: container.client
+         }
+       end}
+    end
```

Comment:
Any function that return a list of object are breaking changes since version 0.9.7 but didn't notice user about this breaking changes. Do we need to add a changelog about this?